code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm

```diff
@@ -704,6 +704,50 @@
 
 	spawn_meteors(ramp_up_final, wavetype)
 
+/// Ruleset for thieves
+/datum/dynamic_ruleset/roundstart/thieves
+	name = "Thieves"
+	antag_flag = ROLE_THIEF
+	antag_datum = /datum/antagonist/thief
+	protected_roles = list(
+		JOB_CAPTAIN,
+		JOB_DETECTIVE,
+		JOB_HEAD_OF_SECURITY,
+		JOB_PRISONER,
+		JOB_SECURITY_OFFICER,
+		JOB_WARDEN,
+	)
+	restricted_roles = list(
+		JOB_AI,
+		JOB_CYBORG,
+	)
+	required_candidates = 1
+	weight = 5
```

Comment:
I don't think anything should be at the same weight as traitors, you should drop this down to like, 3.

```suggestion
	weight = 3
```

Reply:
agreed